### DISCUSSION TITLE
Aggiunta funzione di padding nei template fattura

### DISCUSSION THREAD

**Discussion by Alexxia5** [fattureincloud/api#430] [Q&A] [answered]
Ciao,
 nella documentazione per la personalizzazione dei template delle fatture ho visto che si possono applicare alcune funzioni ai campi ad es: brightness, uppercase, lowercase, ecc... ma non c'è nessuna funzione che permette di effettuare il padding di un campo.
Nello specifico avrei la necessità di visualizzare il numero della fattura con 6 digit totali con un padding di zeri a sinistra.
Ad es se il numero fattura è 23 nella fattura dovrei visualizzare 000023. Nel template mi permette di flaggare una visualizzazione fino a 3 byte, potendo quindi visualizzare 023, ma non 000023 come serve a me.

Esiste altrimenti un modo alternativo per otterere ciò che mi serve?
Grazie mille.
Alex

**Comment by emajo** [comment, marked as answer]
Ciao @Alexxia5, per queste domande più legate al prodotto consiglio di scrivere direttamente al supporto clienti in quanto sono più specializzati su questo tipo argomenti.

Grazie
Emanuele.

**Reply by Alexxia5** [reply]
Per contattarli scrivo all'indirizzo di posta info@fattureincloud.it ?
Oppure esiste un altro gruppo di discussione come questo?
Grazie

**Reply by valmoz** [reply]
@Alexxia5 corretto, deve usare quella mail oppure la chat presente sulla pagina di Fatture in Cloud.
chiudo questa discussione, si senta libera di riaprirla nel caso dovesse servirle altro.

Grazie
Mauro
